diff -urN aa/drwright-0.18/configure.in drwright-0.18/configure.in
--- aa/drwright-0.18/configure.in	2004-09-28 11:14:59.000000000 +0200
+++ drwright-0.18/configure.in	2004-10-03 14:26:31.912389264 +0200
@@ -107,7 +107,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", "")
 
-ALL_LINGUAS="az bg ca cy cs da de el en_CA en_GB es fr gu hr is it ja ko ml mn ms nl no pl pt pt_BR ru sk sq sr sr@Latn sv tr uk zh_CN zh_TW"
+ALL_LINGUAS="az bg ca cy cs da de el en_CA en_GB es fr gu hr is it ja ko ml mn ms nb nl pl pt pt_BR ru sk sq sr sr@Latn sv tr uk zh_CN zh_TW"
 
 AM_GLIB_GNU_GETTEXT
 
